### DISCUSSION TITLE
slideNext with Vue (using swiper methods in vue)

### DISCUSSION THREAD

**Discussion by MentalGear** [nolimits4web/swiper#4226] [Q&A] [answered]
Vue Guide.
I'm wondering how to trigger methods on a vue/slider instance, like slideNext? I couldn't find an example in the guide on it.

**Comment by nolimits4web** [comment, marked as answer]
In the very first usage example https://swiperjs.com/vue#usage you can see how to obtain the Swiper instance that you can use to call methods on

**Reply by MentalGear** [reply]
I noticed the instance code, but overlooked that you saved it into vue's data property to use later. :)
Thanks for the swift reply! Tipped you via patreon (sub/unsub since no one time tips available).

**Reply by nolimits4web** [reply]
Thank you!

**Reply by MentalGear** [reply]
Kudos on using the new discussions feature. Really appreciate your quick support on issues. You may want to add a signature to your replies with a tip or constribute option (the easier to find > more people use it to honor your work).
